tests/plugins/icos/test_reader.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import annotations
+
+from pathlib import Path
+
+import pytest
+
+from pyaerocom import const
+from pyaerocom.plugins.mep.reader import ReadMEP
+from pyaerocom.plugins.icos.reader import ReadICOS
+from tests.conftest import lustre_unavail
+
+try:
+    ICOS_PATH = Path(const.OBSLOCS_UNGRIDDED[const.ICOS_NAME])
+except KeyError:
+    pytest.mark.skip(reason=f"ICOS path not initialised due to non existence in CI")
+
+VARS_DEFAULT = {"vmrco2"}
+VARS_PROVIDED = VARS_DEFAULT  # | {} add more if ever needed
+
+STATION_NAMES = ("bir", "gat", "hpb")
+
+
+@lustre_unavail
```

Comment:
Decorating with `pytest.mark.skipif` only works for test functions.
Decorating a fixture will not mark the tests that use the fixture.

Reply:
#910